stories/core/Table.stories.tsx

```diff
@@ -139,9 +139,6 @@ export const Basic: Story<TableProps> = (args) => {
 
 export const Selectable: Story<TableProps> = (args) => {
   const { columns, data, ...rest } = args;
-  const onClickHandler = (
-    props: CellProps<{ name: string; description: string }>,
-  ) => action(props.row.original.name)();
 
```

Comment:
Maybe add `onRowClick` instead. Also, write above the table that with `Ctrl` you can select multiple rows.

Reply:
Is `onRowClick` really necessary? This is the main story for `Selectable` Table so `onSelect` makes more sense here.

As for the `Ctrl` part, I am hesitant to advertise that explicitly because we don't support `Shift` yet.

Reply:
It is a new prop/functionality so we need to show it somewhere.

Reply:
What if I leave `onSelect` and also add `onRowClick`?